src/sidebar/services/annotations.ts

```diff
@@ -337,4 +337,16 @@ export class AnnotationsService {
 
     return savedAnnotation;
   }
+
+  /**
+   * Fetch an annotation from the API and add or update it in the store
+   */
+  async loadAnnotation(annotationId: string): Promise<Annotation> {
```

Comment:
```suggestion
  async loadAnnotation(id: string): Promise<Annotation> {
```

I think it is clear enough from the context what kind of ID this is.